### ISSUE TITLE
Button into Note (how to hack an HTML field).

### ISSUE THREAD

**Issue by Linekio** [odoo/odoo#33435]
Hi, a user copied pasted something in a note and it has weird effects.

Impacted versions: 11.0, 12.0, more ?

Steps to reproduce:
Copy / paste an Odoo button into an HTML field, and click on it.

Current behavior:
Client error
Expected behavior:
Nothing
Video/Screenshot link (optional):
Vidéo : http://www.k-upload.fr/afficher-video-2019-05-16-164f4b6a8out.mp4.html

**Comment by pedrobaeza** [comment]
That's logical, as you are copying elements that has interactions in full web client. Simply don't copy that...

**Comment by Yenthe666** [comment]
Exactly. I really don't consider this an issue. How on earth do you expect copying half a page of source code with links in it to work dynamically in another page?

**Comment by Linekio** [comment]
I think you can't tell your customer "do not do that".
Especially that it would be enough to catch the events on each HTML fields to prevent this error message.

**Comment by pedrobaeza** [comment]
That's an overhead that doesn't worth IMO.

**Comment by Yenthe666** [comment]
That would create a technical debt and would be focusing on non-essential things.
